assets/src/dashboard/storybookUtils/ampStoryWrapper.js

```diff
@@ -19,9 +19,17 @@
  */
 import PropTypes from 'prop-types';
 
+/**
+ * Internal dependencies
+ */
+import Boilerplate from '../../edit-story/output/utils/ampBoilerplate';
+import CustomCSS from '../../edit-story/output/utils/styles';
+
 function AMPStoryWrapper({ children }) {
   return (
     <div style={{ width: '100%', height: '640px' }}>
+      <Boilerplate />
+      <CustomCSS />
```

Comment:
Adding these styles to our Story Book AMP Stories so they'll have safety zones and danger zones!